AGENTS.md

```
@@ -0,0 +1,149 @@
+
+# Install Codex prompts to ~/.codex/prompts
+pnpm prompts:install
+
+# Force overwrite existing prompts
+pnpm prompts:install -- --force
+```
+
+### Testing
+
+No test suite configured. When added:
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)